cmd/broker/filter/main.go

```diff
@@ -54,8 +55,9 @@ const (
 )
 
 type envConfig struct {
-	Broker        string `envconfig:"BROKER" required:"true"`
-	Namespace     string `envconfig:"NAMESPACE" required:"true"`
+	Broker    string `envconfig:"BROKER" required:"true"`
+	Namespace string `envconfig:"NAMESPACE" required:"true"`
+	// TODO: change this environment variable to something like "PodGroupName".
 	PodName       string `split_words:"true" required:"true"`
```

Comment:
I would update this to be in the form of the above members:  ``envconfig:"POD_NAME" required:"true"`` etc, I think without those explicit values, envconfig is hard to use and find usage.